src/mdlDocumentBuilder/controller.ts

```diff
@@ -21,87 +22,95 @@ import { isErrorCode } from "../utils/isErrorCode";
 import { ERROR_CHOICES } from "../utils/errorChoices";
 import { getTimeToLiveEpoch } from "../utils/getTimeToLiveEpoch";
 import { getRandomIntInclusive } from "../utils/getRandomIntInclusive";
+import { ExpressRouteFunction } from "../types/ExpressRouteFunction";
 
 const CREDENTIAL_TYPE = CredentialType.MobileDrivingLicence;
 const TTL_MINUTES = 43200;
 
 let drivingLicenceNumber: string;
 
-export async function mdlDocumentBuilderGetController(
-  req: Request,
-  res: Response,
-): Promise<void> {
-  try {
-    const { defaultIssueDate, defaultExpiryDate } = getDefaultDates();
-    drivingLicenceNumber = "EDWAR" + getRandomIntInclusive() + "SE5RO";
-    res.render("mdl-document-details-form.njk", {
-      defaultIssueDate,
-      defaultExpiryDate,
-      drivingLicenceNumber,
-      authenticated: isAuthenticated(req),
-      errorChoices: ERROR_CHOICES,
-    });
-  } catch (error) {
-    logger.error(
-      error,
-      "An error happened rendering Driving Licence document page",
-    );
-    res.render("500.njk");
-  }
+export interface MdlDocumentBuilderControllerConfig {
+  environment?: string;
 }
 
-export async function mdlDocumentBuilderPostController(
-  req: Request,
-  res: Response,
-): Promise<void> {
-  try {
-    const body: MdlRequestBody = req.body;
-
-    const errors = validateDateFields(body);
-    if (Object.keys(errors).length > 0) {
+export function mdlDocumentBuilderGetController({
+  environment = getEnvironment(),
+}: MdlDocumentBuilderControllerConfig = {}): ExpressRouteFunction {
+  return async function (req: Request, res: Response): Promise<void> {
+    try {
+      const showThrowError = environment !== "staging";
       const { defaultIssueDate, defaultExpiryDate } = getDefaultDates();
-      return res.render("mdl-document-details-form.njk", {
+      drivingLicenceNumber = "EDWAR" + getRandomIntInclusive() + "SE5RO";
+      res.render("mdl-document-details-form.njk", {
         defaultIssueDate,
         defaultExpiryDate,
         drivingLicenceNumber,
         authenticated: isAuthenticated(req),
         errorChoices: ERROR_CHOICES,
-        errors,
+        showThrowError,
       });
+    } catch (error) {
+      logger.error(
+        error,
+        "An error happened rendering Driving Licence document page",
+      );
+      res.render("500.njk");
     }
+  };
+}
```

Comment:
```suggestion
}

```
Add back the empty line between functions